extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -67,6 +67,8 @@ private static function addOoyalaAssets(\OutputPage $out, $featuredVideoData, $r
 	}
 
 	private static function addJWPlayerAssets() {
+		// fixme that causes extra request
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)

[project-id]: wikia:app